gfx/IsoMSX+/tile_info.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "pixelscale": 1,
+      "height": 16,
+      "iso": true,
+      "width": 32
+  },
+  {
+    "furniture.png": { "sprite_width": 32, "sprite_height": 32 }
```

Comment:
Probably should name tilesheet according to their size, we might make 32x32 sprite that are not furnitures at some point for exemple

Reply:
I based this on the MShockXotto+ repository, where the sprite folders are named pngs_tiles_32x32 or pngs_tall _32x64 but the tilesheets they compile into are simply noted as tiles.png or tall.png in the tile_info.json, omitting the specified sprite size.

Is this naming convention not necessary for the tileset compiler to work?